.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,6 +12,7 @@ For example, instead of "Fixes #1234" use "Introduce new method to calculate 1+1
 <!-- Put an `x` in all the boxes that apply. -->
 <!-- If your change requires a documentation PR, please link it appropriately -->
 <!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+<!-- Feel free to remove irrelevant items. -->
```

Comment:
I agree that this is debatable, but if you just leave them unchecked there is not difference between “not done yet” and “not applicable”. (And it can interfere with the use of actual tasks list in the rest of the issue, as github will show the corresponding items as remaining tasks.) In any case, this is just a detail, and I'm fine with dropping this commit if you insist.

Reply:
I don't have any strong feelings about this either.